Orange/base.py

```diff
@@ -18,13 +19,12 @@ class Learner:
     name = 'learner'
     #: A sequence of data preprocessors to apply on data prior to
     #: fitting the model
-    preprocessors = ()
+    preprocessors = None
     learner_adequacy_err_msg = ''
 
     def __init__(self, preprocessors=None):
-        if preprocessors is None:
-            preprocessors = type(self).preprocessors
-        self.preprocessors = list(preprocessors)
+        self.use_default_preprocessors = False
```

Comment:
You're right of course, this deserves explanation. I would like to address the reason I didn't add `use_default_preprocessors` here as a constructor argument.

The reason for this was purely practical. Every learner reimplements the `__init__` method and they only specify which arguments that specific learner needs + preprocessors ([example](https://github.com/biolab/orange3/blob/master/Orange/classification/logistic_regression.py#L42-L44)). Since `owleanerwidget` would then call the learner constructor with this new argument, we would have to fix every learner implementation to support this argument as well, exactly like every learner handles the `preprocessor` argument. Since this argument would only ever be used in rare cases, I don't think it warrants changing every learner and adding that additional argument, since then each learner constructor becomes more complicated, when it should really only care about its own specific arguments and not about the underlying learner logic.